tests/unit-tests/polyfills/functions.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
```

Comment:
> I am in favor of forcing Beans to be in a WP environment for pure Unit Tests though.

Unless you've missed a "not" in your sentence, this would appear to be contradictory. Unit tests should be runnable without any DB, any file system, and any other dependencies like WP.

Reply:
@GaryJones I did miss "not" in my sentence indeed ;-)